frontend/core-ui/src/modules/products/detail/components/tagsManager.tsx

```
@@ -119,6 +130,14 @@ export function TagsManager({
     });
   };
 
+  // Context provider for the CreateTagForm
+  const contextValue = {
+    newTagName: '',
+    tagType: 'core:product',
+    onSelect: handleTagCreated,
+    setNewTagName: () => {},
```

ESLint

[error] Unexpected empty method 'setNewTagName'.

(@typescript-eslint/no-empty-function)